### PR TITLE
Update name of package for newer version

### PR DESCRIPTION
**Pull request by jonathan-s**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by teizenman** [comment]
@jonathan-s 
Why would you want to change the display name of the integration?
This can cause confusion among users. It is recommended to use the same name.

**Comment by jonathan-s** [comment]
Hi @teizenman, 
That was a request I got from one of my colleagues at Recorded Future where I work to reflect that the version has in fact been upgraded, rather than just saying that this is version 2 of the plugin. 

I'm also noting that in the release notes it still says that it's version 1. Perhaps it would be better to reflect the change there. For context, we recently merged a PR where there were quite a lot of changes going in. So it would be good to let users know that there has been an update by signaling the version number change which is the intention behind this PR.

**Comment by teizenman** [comment]
This is what the marketplace is for, users can see the version they are using, they will have to download and upgrade the current version in order to have it running.

This is what it will look like for user that did not yet upgrade to the newest version:
image

As you can see - they will get the notification to upgrade, and what has chagned.

File: Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ commonfields:
   id: Recorded Future v2
   version: -1
 name: Recorded Future v2
-display: Recorded Future v2
+display: Recorded Future v2.1
```
Comment (ShahafBenYakir) [inline Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml:5]: @jonathan-s why have we decided to do this change? The `v2` refers to the difference between the previous (XSOAR supported) pack to this one. I think the addition `.1` could confuse customers with the real pack version (which shouldn't appear in the display name).

**Comment by jonathan-s** [comment]
I've discussed with my colleague and he's agreed to leave it as is.

**Pull request closed without merging**